cmd/valet_link.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"trellis-cli/trellis"
+)
+
+type ValetLinkCommand struct {
+	UI      cli.Ui
+	Trellis *trellis.Trellis
+}
+
+func (c *ValetLinkCommand) Run(args []string) int {
+	if err := c.Trellis.LoadProject(); err != nil {
+		c.UI.Error(err.Error())
+		return 1
+	}
+
+	var environment string
+
+	switch len(args) {
+	case 0:
+		environment = "development"
+	case 1:
+		environment = args[0]
+	default:
+		c.UI.Error(fmt.Sprintf("Error: too many arguments (expected 0 or 1, got %d)\n", len(args)))
+		c.UI.Output(c.Help())
+		return 1
+	}
+
+	config, ok := c.Trellis.Environments[environment]
+	if !ok {
+		c.UI.Error(fmt.Sprintf("Error: %s is not a valid environment", environment))
+		return 1
+	}
+
+	c.UI.Info(fmt.Sprintf("Linking environment %s...\n", environment))
```

Comment:
Based on your sample output, I might remove this newline and add one after "Linking site"?

Reply:
New output after https://github.com/roots/trellis-cli/pull/48/commits/c047d5b7a1dcbd8c8dc8e2f069df838dcfec5463:

```sh-session
$ trellis-cli valet link
Linking environment development...
Linking site no-enabled.com...

Running command => valet link no-enabled
A [no-enabled] symbolic link has been created in [/Users/me/.config/valet/Sites/no-enabled].
✓ Site no-enabled.com linked

Linking site insecure.com...

Running command => valet link insecure
A [insecure] symbolic link has been created in [/Users/me/.config/valet/Sites/insecure].
✓ Site insecure.com linked
```

Am I understanding it correctly?

Reply:
Yep looks great to me 👍